react-style-guide.md

```diff
@@ -0,0 +1,167 @@
+# React Style Guide
+
+- [High Level Structure](#high-level-structure)
+- [Folder and File Names](#folder-and-file-names)
+- [Components](#components)
+- [Imports & Exports](#imports--exports)
+- [Routing](#routing)
+- [Forms](#forms)
+- [???](#???)
+
+
+## High Level Structure
+
+### Feature Folders
+The project should be organised into **feature folders**. If you are building a banking system, then your project file structure could look like this:
+```
+- src/
+  - accounts/
+  - core/
+  - onboarding/
+  - payments/
+  - index.js
+```
+Each feature folder will contain folders that group files by type (components, pages, models/entities, services, utils, etc.).
+```
+- src/
+  - accounts/
+    - components/
+    - pages/
+    - redux/
```

Comment:
Is this Redux folder needed if we have core redux and a redux file inside the component itself?